generator/golang/backend.go

```diff
@@ -265,3 +269,23 @@ func (g *GoBackend) PostProcess(path string, content []byte) ([]byte, error) {
 	}
 	return content, nil
 }
+
+func (g *GoBackend) removeStreamingFunctions(ast *parser.Thrift) {
+	for _, svc := range ast.Services {
+		functions := make([]*parser.Function, 0, len(svc.Functions))
+		for _, f := range svc.Functions {
+			st, err := streaming.ParseStreaming(f)
+			if err != nil {
+				g.log.Warn(fmt.Sprintf("%s.%s: failed to parse streaming, err = %v", svc.Name, f.Name, err))
+				continue
+			}
+			if st.IsStreaming {
+				g.log.Warn(fmt.Sprintf("skip streaming function %s.%s: not supported by your kitex, "+
+					"please update to the latest kitex", svc.Name, f.Name))
```

Comment:
please upgrade kitex tool?

Reply:
“please update your kitex tool to the latest version”

Reply:
done